common/buildcraft/builders/tile/TileQuarry.java

```diff
@@ -635,6 +673,20 @@ public final boolean addPower(long microJoules) {
                 return onReceivePower();
             }
         }
+
+        public List<ItemStack> pickupItems(BlockPos pos) {
+            List<ItemStack> stacks = new ArrayList<>();
+            for (Entity entity : world.getEntitiesWithinAABB(Entity.class, new AxisAlignedBB(pos).expandXyz(1))) {
+                if (entity instanceof EntityPlayer)
+                    continue;
```

Comment:
````java
for (EntityItem entity : world.getEntitiesWithinAABB(EntityItem.class, new AxisAlignedBB(pos).expandXyz(1))) {
  // Rest of code
}
````
As we don't want to be stealing items from anything other than a dropped item.

This could also just call ````InventoryUtil.addToBestAcceptor```` manually, as that's what all callers do (no need to add to a list).